tomography_preprocessing/tilt_series_alignment/_job_utils.py

```diff
@@ -1,20 +1,28 @@
 from pathlib import Path
-from typing import Tuple
+from typing import Tuple, Optional
 
 import numpy as np
 import pandas as pd
+import starfile
+import typer
 
+from .imod._utils import get_tilt_series_alignment_data, calculate_specimen_shifts, \
+    get_xyz_extrinsic_euler_angles
 from ..utils.transformations import S, Rx, Ry, Rz
+from ._cli import cli
 
 
-def create_alignment_job_directory_structure(output_directory: Path) -> Tuple[Path, Path]:
-    """Create directory structure for an IMOD alignment job."""
+def create_alignment_job_directory_structure(output_directory: Path) -> Tuple[Path, Path, Path]:
+    """Create directory structure for a tilt-series alignment job."""
     tilt_series_directory = output_directory / 'tilt_series'
     tilt_series_directory.mkdir(parents=True, exist_ok=True)
```

Comment:
This creates a 'tilt series' directory inside the main job directory. Do you want it here, or inside the externals directory?